builder/Dockerfile

```diff
@@ -6,8 +6,12 @@ RUN CGO_ENABLED=0 GOOS=linux go build -mod=vendor github.com/Azure/aks-periscope
 
 FROM alpine
 RUN apk --no-cache add ca-certificates
+RUN apk add --no-cache curl openssl bash

```

Comment:
could we merge lines 8 and 9?

Reply:
💡 thanks, sounds kuch better, If we are going to use helm package we will remove docker file addition: https://github.com/Azure/aks-periscope/pull/54#discussion_r636775650

Or shall we make this part of our refactor, although helm being new addition worth using the helm package.

Reply:
💡 @sophsoph321 lets make this change and then please double check the checklist, and we should plan to merge this.

One-liner will become : (Line 8, 9)

* `RUN apk --no-cache add ca-certificates curl openssl bash`

Reply:
@Tatsinnit,  I have fixed this change in my most recent push. I have reviewed the checklist and have tested my changes locally along with running the CI